test/CMakeLists.txt

```diff
@@ -86,11 +86,17 @@ if( iodaconv_bufr_ENABLED )
     testinput/gdas.t00z.1bhrs4.tm00.bufr_d
```

Comment:
Line 84 (two lines above, I can not add a comment there) lists testinput/bufr_test.yaml, but that file does not seem exist.

Reply:
Good catch! I removed it.